setup.py

```diff
@@ -29,8 +27,6 @@
           'Topic :: Multimedia :: Graphics :: 3D Rendering',
           'Topic :: Scientific/Engineering :: Visualization',
     ],
-    install_requires=['jupyter', 'vpython'],
-    #cmdclass=cmdclass('vpnotebook/data'),
-    #package_data={'vpython': ['data/*'], 'vpnotebook': ['data/kernel.json']},
+    install_requires=['jupyter', 'vpnotebook'],
```

Comment:
Oops. It's gone from vpython.py, but it's still in rate_control.py ;-(. Sorry, I missed that.

Bruce, should I work on removing numpy from rate_control? Is that something you intend to have done by someone?

thx.
-steve